openscenario/openscenario_interpreter/src/syntax/relative_object_position.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <geometry/transform.hpp>
+#include <openscenario_interpreter/reader/attribute.hpp>
+#include <openscenario_interpreter/reader/element.hpp>
+#include <openscenario_interpreter/syntax/relative_object_position.hpp>
+
+namespace openscenario_interpreter
+{
+inline namespace syntax
+{
+RelativeObjectPosition::RelativeObjectPosition(const pugi::xml_node & node, Scope & scope)
+: orientation(readElement<Orientation>("Orientation", node, scope)),
+  reference(readAttribute<EntityRef>("entityRef", node, scope)),
```

Comment:
```suggestion
  entity_ref(readAttribute<String>("entityRef", node, scope)),
```

Reply:
I reffered [relative_world_position](https://github.com/tier4/scenario_simulator_v2/blob/master/openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/relative_world_position.hpp#L47) and if changing the name to `entity_ref` we need to change relative_worl_position and 
[applyTeleportAction](https://github.com/tier4/scenario_simulator_v2/blob/master/openscenario/openscenario_interpreter/src/syntax/teleport_action.cpp#L64) too. Is it better?

Reply:
I understand why you used the data member name `reference`.

The `EntityRef` type data member name for `RelativeWorldPosition` being `reference` is odd given the current style of implementation of syntax-related types. I checked and the code in question was [very old code implemented by me on 12/04/2020](https://github.com/tier4/scenario_simulator_v2/commit/30aa81d7c958bb02cc2461e215a1a1071a354e88). Perhaps at that time I had not defined a (private) rule that the structure of syntax-related classes should match well with the OpenSCENARIO standard.

Therefore, the answer to your question is "yes, that is better".

Reply:
ok, thanks! Fixed in https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/0dfdb61c8df0b18a1e384b50f58a1d22562bfe5b